### PR TITLE
Makes tickets stop roundend, and ping admins if unclaimed

### LINKED ISSUE #630: The round ends, regardless of if there are open tickets or not.

### PR DESCRIPTION
**Pull request by Amelia0010**
fixes: #630 

Interval starts at 60, then drops down to 55, 45, 40, 35, 30, 25, 20, 15, 10 seconds.

### CODE REVIEW

**Comment by Altoids1** [comment]
Bless you

**Comment by Amelia0010** [comment]
This was done badly.

**Comment by Amelia0010** [comment]
I realize it's easier to work off this PR than it is to remake it

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -599,6 +599,9 @@ SUBSYSTEM_DEF(ticker)
 	if(delay_end && !skip_delay)
 		to_chat(world, "<span class='boldannounce'>An admin has delayed the round end.</span>")
 		return
+	if(GLOB.ahelp_tickets.ticketAmount) //YOGS - tickets
```
Comment (ThatLing) [inline code/controllers/subsystem/ticker.dm:602]: What if there's no admins online 🤔

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (Amelia0010) [reply code/controllers/subsystem/ticker.dm:602]: oh yeaaah

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -599,6 +599,9 @@ SUBSYSTEM_DEF(ticker)
 	if(delay_end && !skip_delay)
 		to_chat(world, "<span class='boldannounce'>An admin has delayed the round end.</span>")
 		return
+	if(GLOB.ahelp_tickets.ticketAmount && GLOB.admins.len) //YOGS - tickets
```
Comment (ThatLing) [inline code/controllers/subsystem/ticker.dm]: Make it show the "theres still tickets but no admins, make a complaint on the forum" message.

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by ThatLing — APPROVED** [review]

**Pull request merged**